test/staging/Intl402/Temporal/old/date-time-format.js

```diff
@@ -518,6 +537,15 @@ assert.deepEqual(us.formatToParts(t.withCalendar(usCalendar).toPlainYearMonth())
     value: "2020"
   }
 ]);
+
+// Work around variability in month format caused by https:// unicode-org.atlassian.net/browse/CLDR-16243
+var deMonthInYearMonthHasLeadingZero = new Intl.DateTimeFormat("de-AT", {
+  year: "numeric",
+  month: "numeric"
+})
+  .formatToParts(new Date(2000, 3, 1))
+  .find((part) => part.type === "month")
+  .value.startsWith("0");
```

Comment:
This appears to be unused.
```suggestion
```

Reply:
Whoops, looks like I lost this check during the last rebase. The next line should have used this value in a conditional:
```js
assert.deepEqual(at.formatToParts(t.withCalendar(atCalendar).toPlainYearMonth()), [
  {
    type: "month",
    value: deMonthInYearMonthHasLeadingZero ? "02" : "2"
  },
```

Will be fixed in the next push. 

It's actually lucky you found this, because it looks like something went wrong on the proposal-temporal side of this PR and it wasn't picking up the right Test262 subproject commit. Otherwise the proposal-temporal PR should have failed. Investigating that now.  Thanks!